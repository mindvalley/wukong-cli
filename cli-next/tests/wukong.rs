mod common;

#[test]
fn test_wukong_help_without_config_file() {
    let cmd = common::wukong_raw_command()
        .arg("help")
        .env("WUKONG_DEV_CONFIG_FILE", "path/to/non/exists/config.toml")
        .assert()
        .success();

    let output = cmd.get_output();

    insta::assert_snapshot!(std::str::from_utf8(&output.stdout).unwrap());
}

#[test]
fn test_wukong_version_without_config_file() {
    let cmd = common::wukong_raw_command()
        .arg("--version")
        .env("WUKONG_DEV_CONFIG_FILE", "path/to/non/exists/config.toml")
        .assert()
        .success();

    let output = cmd.get_output();

    insta::assert_snapshot!(std::str::from_utf8(&output.stdout).unwrap());
}
