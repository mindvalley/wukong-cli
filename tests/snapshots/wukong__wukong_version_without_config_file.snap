---
source: tests/wukong.rs
expression: "std::str::from_utf8(&output.stdout).unwrap()"
---
wukong 1.0.1

